### PR TITLE
Add test ids

### PR DESCRIPTION
**Pull request by ILpinto**
Add test ids
```
None
```

### CODE REVIEW

File: tests/nodeLabeller_test.go [hunk 1]
```diff
@@ -172,11 +172,11 @@ var _ = Describe("Node Labeller", func() {
...
 
 		table.DescribeTable("should restore modified app labels", expectAppLabelsRestoreAfterUpdate,
-			table.Entry("cluster role", &clusterRoleRes),
-			table.Entry("cluster role binding", &clusterRoleBindingRes),
-			table.Entry("security context constraint", &securityContextConstraintRes),
-			table.Entry("Config Map", &configMapRes),
-			table.Entry("daemonSet", &daemonSetRes),
+			table.Entry("[test_id:5818] cluster role", &clusterRoleRes),
+			table.Entry("[test_id:5819] cluster role binding", &clusterRoleBindingRes),
+			table.Entry("[test_id:CNV-5820] security context constraint", &securityContextConstraintRes),
```
Comment (kwiesmueller) [inline tests/nodeLabeller_test.go:177]: Looks like you dropped a CNV- in here :-)

**Review by kwiesmueller — CHANGES_REQUESTED** [review]
I think some IDs are still missing (mainly on creation): https://github.com/kubevirt/ssp-operator/pull/128/files#diff-2cff42e0947060c1c15b79a4a5f64bee443c04444b3b9a838e3add74ecbad9b6R130

File: tests/nodeLabeller_test.go (hunk 1 above)
Comment (ILpinto) [reply tests/nodeLabeller_test.go]: Thanks @kwiesmueller 
Fix both also in validator_test.go

File: tests/nodeLabeller_test.go [hunk 2]
```diff
@@ -19,7 +19,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-var _ = Describe("Node Labeller", func() {
+var _ = Describe("[rfe_id:4352]Node Labeller", func() {
```
Comment (ksimon1) [inline tests/nodeLabeller_test.go:22]: isn't it missing these tags? [crit:medium][vendor:cnv-qe@redhat.com][level:component]

File: tests/validator_test.go [hunk 3]
```diff
@@ -22,7 +22,7 @@ import (
 	validator "kubevirt.io/ssp-operator/internal/operands/template-validator"
 )
 
-var _ = Describe("Template validator", func() {
+var _ = Describe("[rfe_id:4352]Template validator", func() {
```
Comment (ksimon1) [inline tests/validator_test.go:25]: same here

**Comment by kwiesmueller** [comment]
@ILpinto the test IDs still seem incomplete for the app labels, see: https://github.com/kubevirt/ssp-operator/pull/128#pullrequestreview-606410778
Could you add them?

**Comment by ILpinto** [comment]
> @ILpinto the test IDs still seem incomplete for the app labels, see: [#128 (review)](https://github.com/kubevirt/ssp-operator/pull/128#pullrequestreview-606410778)
> Could you add them?

It fixed already, can you review?

**Comment by kwiesmueller** [comment]
I was looking at https://github.com/kubevirt/ssp-operator/pull/128/files#diff-da6fdef8f8602ed41fa46a470f823e4b4c53d37a9cc70f7e30098050562fe7c6R127 which still needs labels. Or am I looking at an old commit?

**Comment by omeryahud** [comment]
@ILpinto Hi Israel, can you please rebase this?

**Comment by kwiesmueller** [comment]
Can you add the test_ids for the `create app label` tests as well?
e.g. https://github.com/kubevirt/ssp-operator/pull/128/files#diff-2cff42e0947060c1c15b79a4a5f64bee443c04444b3b9a838e3add74ecbad9b6R137

**Comment by ILpinto** [comment]
> Can you add the test_ids for the `create app label` tests as well?
> e.g. https://github.com/kubevirt/ssp-operator/pull/128/files#diff-2cff42e0947060c1c15b79a4a5f64bee443c04444b3b9a838e3add74ecbad9b6R137
@kwiesmueller 
I want to sync with Omer and see what cases he has from Ruth, and also go over Polarion and check we don't have duplicate cases here and in Kubevirt after the remove of node labeller. 
We can do other round of test ids update for sure ... 
So i prefer to add the test ids in this patch and do one more cycle after the checks.

WYT?

**Comment by vatsalparekh** [comment]
@ILpinto Can you please add release note here in the PR description

**Pull request closed without merging**